Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PYTHON_VERSION=3.7
```

Comment:
[question] Do we have any tests to check this change is ok?

Reply:
Ah, `docker build -f ../Dockerfile .` is supposed to be the correct way. I could not find how to use this file.

Reply:
I'm afraid that python 3.7 and 3.8 were not working as I expected
 
### 3.7
```bash
#9 46.19 ERROR: Could not find a version that satisfies the requirement torchvision==0.9.0 (from optuna[checking,doctest,document,testing]) (from versions: 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.2.2.post2, 0.2.2.post3, 0.9.1, 0.10.0, 0.11.0, 0.11.1)
#9 46.19 ERROR: No matching distribution found for torchvision==0.9.0
------
executor failed running [/bin/sh -c if [ "${BUILD_TYPE}" = "dev" ]; then         if [ "${PYTHON_VERSION}" \< "3.6" ]; then             pip install ${PIP_OPTIONS} -e '.[doctest, document, testing]' -f https://download.pytorch.org/whl/torch_stable.html;         else             pip install ${PIP_OPTIONS} -e '.[checking, doctest, document, testing]' -f https://download.pytorch.org/whl/torch_stable.html;         fi     else         pip install ${PIP_OPTIONS} -e .;     fi     && pip install ${PIP_OPTIONS} jupyter notebook]: exit code: 1
```

### 3.8
```bash
#9 49.48 ERROR: Could not find a version that satisfies the requirement tensorflow (from optuna[checking,doctest,document,testing]) (from versions: none)
#9 49.48 ERROR: No matching distribution found for tensorflow
------
executor failed running [/bin/sh -c if [ "${BUILD_TYPE}" = "dev" ]; then         if [ "${PYTHON_VERSION}" \< "3.6" ]; then             pip install ${PIP_OPTIONS} -e '.[doctest, document, testing]' -f https://download.pytorch.org/whl/torch_stable.html;         else             pip install ${PIP_OPTIONS} -e '.[checking, doctest, document, testing]' -f https://download.pytorch.org/whl/torch_stable.html;         fi     else         pip install ${PIP_OPTIONS} -e .;     fi     && pip install ${PIP_OPTIONS} jupyter notebook]: exit code: 1
```

Reply:
I see. Maybe, better to ask the other committers about how they test it.

Reply:
I suppose `.github/workflows/dockerimage.yml` tests it, so it might be okay.

Reply:
I see! Thank you! I will check this PR again soon.